frontend-v2/src/components/review/ReviewCard.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import { BookmarkIcon, HandThumbUpIcon } from "@heroicons/react/24/outline";
+import Rating from "../Rating";
+import TruncatedDescription from "../TruncatedDescription";
+import ReportModal from "./ReportModal";
+
+export type Review = {
+  reviewId: string;
+  authorName: string;
+  title: string;
+  description: string;
+  grade: number;
+  termTaken: string;
+  createdTimestamp: string;
+  updatedTimestamp: string;
+  upvotes: string[];
+  manageability: number;
+  enjoyability: number;
+  usefulness: number;
+  overallRating: number;
+};
+
+export default function ReviewCard({ review }: { review: Review }) {
+  const handleUpvotes = () => {
+    // TODO: Handle upvotes here (Do this when user session can already be handled)
+  };
+
+  const handleBookmark = () => {
+    // TODO: Handle bookmark here (Do this when user session can already be handled)
+  };
+
+  return (
+    <div className="px-10 py-5 space-y-2 shadow-review-card rounded-md bg-white isolate">
+      {/* Title + Date */}
+      <div className="flex items-center gap-2 flex-wrap justify-between">
+        <h1 className="font-bold">{!review.title ? "-" : review.title}</h1>
+        <p>{new Date(review.createdTimestamp).toLocaleDateString()}</p>
```

Comment:
This line seems to be causing issues with react hydration in devtools i.e. the server is expecting date to be formatted as MM/DD/YYYY but we've formatted it to be DD/MM/YYYY. Might be helpful to take a look at [this](https://stackoverflow.com/questions/72673362/error-text-content-does-not-match-server-rendered-html) to resolve them?